test_msgs/msg/DynamicArrayDynamicArrayPrimitivesNested.msg

```diff
@@ -0,0 +1 @@
+DynamicArrayPrimitives[] dynamic_array_dynamic_array_primitive_values
```

Comment:
This case already exists, equivalent to `DynamicArrayPrimitivesNested`:

https://github.com/ros2/rcl_interfaces/blob/2a85883cf64d9dfbe9f15a43966bdb00819aac58/test_msgs/msg/DynamicArrayPrimitivesNested.msg#L1

Reply:
You are right, we missed this one.